test/cql-pytest/cassandra_tests/validation/entities/json_test.py

```diff
@@ -929,6 +924,15 @@ def testToJsonFct(cql, test_keyspace):
             execute(cql, table, "INSERT INTO %s (k, durationval) VALUES (?, P1Y1M2DT10H5M)", 0)
             assert_rows(execute(cql, table, "SELECT k, toJson(durationval) FROM %s WHERE k = ?", 0), [0, "\"1y1mo2d10h5m\""])
 
+# Reproduces issue #7997
+def testTimestampToJson(cql, test_keyspace):
```

Comment:
Please move this test to `cql-pytest/test_json.py` (for reasons I explained above).
Please note that that the coding convention in the general cql-pytest/ directory is different from cassandra_tests - in cassandra_tests I used a bunch of convenience functions to help me port the Java unit tests, but these aren't available and aren't needed in cql-pytest/test_json.py.

But beyond that, you should look at the content of the test and simplify it the bare minimum... This test does NOT need the create_type (I don't know why you needed it), and it shouldn't even create a new table just for this test - it should use the "table1" table that all the tests in the same file are using. This reduces the runtime of such a test from something like 0.1 seconds to something like 0.001 seconds, and makes me happy ;-)